cmd/snap-bootstrap/cmd_initramfs_mounts_test.go

```diff
@@ -3740,7 +3723,7 @@ grade=signed
 	c.Assert(filepath.Join(dirs.SnapBootstrapRunDir, fmt.Sprintf("%s-model-measured", s.sysLabel)), testutil.FilePresent)
 }
 
-func (s *initramfsMountsSuite) TestInitramfsMountsRecoverModeDegradedUnencryptedDataSaveEncryptedUnhappy(c *C) {
+func (s *initramfsMountsSuite) TestInitramfsMountsRecoverModeDegradedUnencryptedDataSaveEncryptedHappy(c *C) {
 	// test a scenario when data is unencrypted but save is encrypted
```

Comment:
this got only the test name changed?  the comment seems to describe an error situation but the test now has Happy in the name? a bit confused

Reply:
I've tweaked the comment here too. It is indeed a happy case when happy == no error from initramfs-mounts. The scenario is a rather impossible one, where data is unencrypted, but ubuntu-save-enc exists. After seeing plain data, we only look for plain save, thus completely ignoring ubuntu-save-enc and there are no further calls to secboot.UnlockVolume*